runtime/common/src/claims.rs

```diff
@@ -48,29 +48,35 @@ pub trait Trait: system::Trait {
 	type Prefix: Get<&'static [u8]>;
 }
 
-/// The kind of a statement this account needs to make for a claim to be valid.
+/// The kind of a statement an account needs to make for a claim to be valid.
 #[derive(Encode, Decode, Clone, Copy, Eq, PartialEq, RuntimeDebug)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 pub enum StatementKind {
-	/// One kind of statement; this is the default.
-	Default,
-	/// Another kind of statement(!).
-	Alternative,
+	/// Statement required to be made by non-SAFT holders.
+	Regular,
+	/// Statement required to be made by SAFT holders.
+	Saft,
 }
 
 impl StatementKind {
 	/// Convert this to the (English) statement it represents.
 	fn to_text(self) -> &'static [u8] {
 		match self {
-			StatementKind::Default => &b"Default"[..],
-			StatementKind::Alternative => &b"Alternative"[..],
+			StatementKind::Regular =>
+				&b"I hereby agree to the terms of the statement whose SHA-256 multihash is \
```

Comment:
Should the hash be described as base58 too?
On what content is it calculated (bytes from the html file, some other textual representation), I naively try to check it by running other the full html file but did not get the same hash?

Reply:
This is the [multihash](https://github.com/multiformats/multibase) calculated from the html, it is the same as what IPFS uses so that the files can be directly accessed using their hash, e.g on https://gateway.pinata.cloud/ipfs/QmXEkMahfhHJPzT3RjkXiZVFi77ZeVeuxtAjhojGRNYckz

Reply:
it's a multihash, as is typical for ipfs. i haven't checked that they actually hash correctly, but i presume @keorn has this verified.

Reply:
Using ipfs hashes checks.